rds-apb/ansibleapp.yml

```diff
@@ -0,0 +1,70 @@
+id: 84e8173e-5ced-4489-9b4d-aac7e779c47e
+name: ansibleplaybookbundle/rds-apb
```

Comment:
I hardcoded the UI to expect 'rds-postgres-apb' when we get around to breaking this out to a postgres apb